AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -971,3 +984,51 @@ - (void)dealloc
 }
 
 @end
+
+@implementation ASMainQueueSerialQueue
```

Comment:
Sure, can you think of a better name while we're at it? I'm not in love with this one.